DependencyInjection/Configuration.php

```diff
@@ -39,7 +39,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('form')->defaultValue('SuluCommentBundle:WebsiteComment:form.html.twig')->end()
                     ->end()
                 ->end()
+                ->booleanNode('nested_comments')->defaultTrue()->end()
```

Comment:
should we enable this per default? will most projects use nested comments?

Reply:
i think there will be both - in my opinion by default disabled features will be never be enabled :)